pkgs/test_core/lib/scaffolding.dart

```diff
@@ -94,7 +94,11 @@ Declarer get _declarer {
 ///
 /// If [skip] is a String or `true`, the test is skipped. If it's a String, it
 /// should explain why the test is skipped; this reason will be printed instead
-/// of running the test.
+/// of running the test. If a [test] is nested within a [group], the
```

Comment:
It is probably worth mentioning the behavior with regards to a suite level `@Skip` as well? I presume neither group/test configuration will take precedence because we never even compile/run the test in that case. But you would have to confirm.

Reply:
I am not seeing any for this specific behavior, for the group/test behavior you could add a test after [this one](https://github.com/dart-lang/test/blob/master/pkgs/test/test/runner/engine_test.dart#L236).

For the `@Skip` handling it has to pretty much be an integration tests, could be next to this one https://github.com/dart-lang/test/blob/master/pkgs/test/test/runner/runner_test.dart#L514.